scripts/qa_crowbarsetup.sh

```diff
@@ -610,9 +643,13 @@ function get_crowbar_node()
     get_all_nodes | grep -v "^d" | head -n 1
 }
 
-function get_sles12_node()
+function get_sles12plus_node()
 {
-    knife search node "target_platform:suse-12.0" -a name | grep ^name: | cut -d : -f 2 | sort | tail -n 1 | sed 's/\s//g'
+    if [ -n "$want_sles12sp1" ]; then
+        knife search node "target_platform:suse-12.1" -a name | grep ^name: | cut -d : -f 2 | sort | tail -n 1 | sed 's/\s//g'
+    else
+        knife search node "target_platform:suse-12.0" -a name | grep ^name: | cut -d : -f 2 | sort | tail -n 1 | sed 's/\s//g'
```

Comment:
this is a bit ugly code duplication - maybe we could always look for suse-12.0 and 12.1 nodes... or make the number variable like
```bash
x=suse-12.0
[[ $want_sles12sp1 ]] && x=suse-12.1
```